### PR TITLE
feat(GlobalHeader): allowing access to Business Account Management on mobile

### PR DESCRIPTION
**Pull request by vonbarown**
### Overview

Allowing the Business Account Management link to appear in mobile view

### PR Checklist

- [x] Related to designs: https://www.figma.com/file/q3c1dXe0ETG1TNVoO0bxNX/Gated-Feature-Upsell-Patterns?node-id=802%3A18602
- [x] Related to JIRA ticket: [EN-1478](https://codecademy.atlassian.net/browse/EN-1478)
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.39-alpha.9c6181.0<br/>@codecademy/gamut-labs@28.6.2-alpha.9c6181.0<br/>@codecademy/styleguide@52.5.1-alpha.9c6181.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://61dc80b69be52b11c2c0feb1--gamut-preview.netlify.app>https://61dc80b69be52b11c2c0feb1--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/61dc80b69be52b11c2c0feb1>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.43-alpha.30a42e.0<br/>@codecademy/gamut-labs@28.6.6-alpha.30a42e.0<br/>@codecademy/styleguide@53.0.4-alpha.30a42e.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://61eae95a8add9f34f824066c--gamut-preview.netlify.app>https://61eae95a8add9f34f824066c--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/61eae95a8add9f34f824066c>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.52-alpha.492d16.0<br/>@codecademy/gamut-labs@29.2.3-alpha.492d16.0<br/>@codecademy/styleguide@54.0.7-alpha.492d16.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://61f93ca6447cc9d87c277199--gamut-preview.netlify.app>https://61f93ca6447cc9d87c277199--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/61f93ca6447cc9d87c277199>Deploy Logs</a></p>

File: packages/gamut-labs/src/GlobalHeader/GlobalHeaderItems.tsx [hunk 1]
```diff
@@ -247,12 +247,9 @@ const profileLogOut: AppHeaderLinkItem = {
...
 
-export const freeProfile = (
-  user: User,
-  isMobile?: boolean
-): AppHeaderProfileDropdownItem => {
+export const freeProfile = (user: User): AppHeaderProfileDropdownItem => {
   const topSection = [profileMyProfile, profileAccount, profileMyHome];
-  if (!isMobile && user.isAccountManager) {
+  if (user.isAccountManager) {
     topSection.push(profileBusinessAccount);
```
Comment (antoniablair) [inline packages/gamut-labs/src/GlobalHeader/GlobalHeaderItems.tsx:253]: should this also be or `user.isBusinessAdmin` ?

File: packages/gamut-labs/src/GlobalHeader/GlobalHeaderVariants.tsx [hunk 2]
```diff
@@ -178,7 +178,7 @@ export const freeMobileHeaderItems = (
     communityDropdown,
     ...(hidePricing ? [] : [pricingDropdown]),
     forBusiness,
-    freeProfile(user, true),
+    freeProfile(user),
     user.showProUpgrade
```
Comment (antoniablair) [inline packages/gamut-labs/src/GlobalHeader/GlobalHeaderVariants.tsx:182]: what are these `true` changes doing? 💭

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.63-alpha.b11a01.0<br/>@codecademy/gamut-labs@29.3.4-alpha.b11a01.0<br/>@codecademy/styleguide@54.3.4-alpha.b11a01.0<br/></pre>

File: packages/gamut-labs/src/GlobalHeader/GlobalHeaderVariants.tsx (hunk 2 above)
Comment (vonbarown) [reply packages/gamut-labs/src/GlobalHeader/GlobalHeaderVariants.tsx:182]: the boolean was checking if was a mobile view

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://620e97f1da339d273b03d5a3--gamut-preview.netlify.app>https://620e97f1da339d273b03d5a3--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/620e97f1da339d273b03d5a3>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.63-alpha.5d5c33.0<br/>@codecademy/gamut-labs@29.3.4-alpha.5d5c33.0<br/>@codecademy/styleguide@54.3.4-alpha.5d5c33.0<br/></pre>

File: packages/gamut-labs/src/GlobalHeader/GlobalHeaderItems.tsx [hunk 3]
```diff
@@ -272,12 +272,9 @@ export const freeProfile = (
...
 };
 
-export const proProfile = (
-  user: User,
-  isMobile?: boolean
-): AppHeaderProfileDropdownItem => {
+export const proProfile = (user: User): AppHeaderProfileDropdownItem => {
   const topSection = [profileMyProfile, profileAccount, profileMyHome];
-  if (!isMobile && (user.isAccountManager || user.isBusinessAdmin)) {
+  if (user.isAccountManager || user.isBusinessAdmin) {
```
Comment (jenesh) [inline packages/gamut-labs/src/GlobalHeader/GlobalHeaderItems.tsx:277]: Just to be safe: 

```suggestion
  if (user?.isAccountManager || user?.isBusinessAdmin) {
```

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://620e9c73fa6b1d326c486e88--gamut-preview.netlify.app>https://620e9c73fa6b1d326c486e88--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/620e9c73fa6b1d326c486e88>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://620ea6edcb14803cb9072a20--gamut-preview.netlify.app>https://620ea6edcb14803cb9072a20--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/620ea6edcb14803cb9072a20>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.64-alpha.375cc5.0<br/>@codecademy/gamut-labs@29.3.5-alpha.375cc5.0<br/>@codecademy/styleguide@54.3.5-alpha.375cc5.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6215055e812c8106372b28d6--gamut-preview.netlify.app>https://6215055e812c8106372b28d6--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6215055e812c8106372b28d6>Deploy Logs</a></p>

**Comment by vonbarown** [comment]
> this all looks good to me! could you add this alpha pkg to portal + monolith PRs so i can test the changes there?

Monolith with alpha package

**Review by dreamwasp — APPROVED** [review]
these changes look great, just wanted to call out that BAM itself is looking a little weird on mobile after i follow the link: 

image

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.97-alpha.7616f7.0<br/>@codecademy/gamut-labs@31.3.2-alpha.7616f7.0<br/>@codecademy/styleguide@56.2.2-alpha.7616f7.0<br/></pre>

**Review by dreamwasp — APPROVED** [review]

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.102-alpha.3633d3.0<br/>@codecademy/gamut-labs@31.6.1-alpha.3633d3.0<br/>@codecademy/styleguide@56.4.1-alpha.3633d3.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://62572b09c1c97c0a0fc250e3--gamut-preview.netlify.app>https://62572b09c1c97c0a0fc250e3--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/62572b09c1c97c0a0fc250e3>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://626163c10645b66ffb64c7d4--gamut-preview.netlify.app>https://626163c10645b66ffb64c7d4--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/626163c10645b66ffb64c7d4>Deploy Logs</a></p>

**Review by dreamwasp — APPROVED** [review]
double approve

**Pull request merged**